package.json

```diff
@@ -35,8 +35,8 @@
     "react": "15.4.1",
     "react-dom": "15.4.1",
     "react-redux": "5.0.1",
-    "react-router": "3.0.0",
-    "react-router-redux": "4.0.7",
+    "react-router-dom": "4.0.0",
+    "react-router-redux": "5.0.0-alpha.3",
```

Comment:
How about replace react-router-redux (which is still in alpha) with [connected-react-router](https://github.com/supasate/connected-react-router) ?